Fable.sln

```diff
@@ -26,14 +26,14 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Files", "Files", "{54BD4E45
 		build.fsx = build.fsx
 	EndProjectSection
 EndProject
-Project("{F2A71F9B-5D33-465A-A702-920D77279786}") = "Fable.Standalone", "src\fable-standalone\src\Fable.Standalone.fsproj", "{D5817BB2-0664-4E0A-BBD1-06615D705E99}"
-EndProject
```

Comment:
I removed Fable.Standalone from the solution to avoid issues [like this one](https://github.com/fable-compiler/Fable/pull/2301#discussion_r532278149). If necessary, we can add a separate solution with fcs-fable + Fable.Standalone.